debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/EventQueue.java

```diff
@@ -238,13 +243,28 @@ class EventDedupCache {
         private static final String QUEUE_FILE_DIR = "queue";
         private long maxCacheSize = 1000000;
         private String currentQueueSegmentPath;
-        private ObjectMapper mapper = new ObjectMapper();
 
         public EventDedupCache(String dataDir) {
             this.dataDir = dataDir;
         }
 
+        private ObjectMapper getObjectMapper() {
```

Comment:
we need to think about not duplicating this code in two places, but okay to defer.